RecoHGCal/TICL/python/EMStep_cff.py

```diff
@@ -51,21 +51,25 @@
 
 filteredLayerClustersHFNoseEM = filteredLayerClustersEM.clone(
     LayerClusters = 'hgcalLayerClustersHFNose',
-    LayerClustersInputMask = "hgcalLayerClustersHFNose:InitialLayerClustersMask",
+    LayerClustersInputMask = 'ticlTrackstersHFNoseTrkEM',
     iteration_label = "EMn",
+    min_cluster_size = 2, # inclusive
     algo_number = 9
-#no tracking mask for EM for now
 )
 
 ticlTrackstersHFNoseEM = ticlTrackstersEM.clone(
     detector = "HFNose",
     layer_clusters = "hgcalLayerClustersHFNose",
     layer_clusters_hfnose_tiles = "ticlLayerTileHFNose",
-    original_mask = "hgcalLayerClustersHFNose:InitialLayerClustersMask",
+    original_mask = "ticlTrackstersHFNoseTrkEM",
     filtered_mask = "filteredLayerClustersHFNoseEM:EMn",
     seeding_regions = "ticlSeedingGlobalHFNose",
     time_layerclusters = "hgcalLayerClustersHFNose:timeLayerCluster",
-    min_layers_per_trackster = 6
+    itername = "EMn",
```

Comment:
Could you use a more expressive name? Also for the downstream `HADn` and `TrkEMn`.